### ISSUE TITLE
Alerts don't get reported in unstable's web interface

### ISSUE THREAD

**Issue by pypt** [munin-monitoring/munin#537]
Hi there,

I have a problem reporting alerts (limits) in Munin's (unstable 2.1.12) HTML interface.

For a home-made plugin that I'm writing, I've set up certain max/min limits for the reported "value". `munin-limits` is able recognize that the max limit is exceeded and report it via `/var/lib/limits` and email. However, the alert is not visible on the web interface (/problems.html and elsewhere).

I think this has something to do with rrdtool + SQLite interaction. `munin-limits` reads the limits straight from `limits.storable`, but the CGI-based HTML page generator [looks into SQLite database for alerts to report](https://github.com/munin-monitoring/munin/blob/devel/lib/Munin/Master/HTML.pm#L106).

If I run the SQL query from the link above directly to the `datafile.sqlite` database, I get a bunch of NULLs back.

Any ideas of what I could try? I've noticed that alerts on [demo page](http://demo.munin-monitoring.org/) work fine.

**Comment by chteuchteu** [comment]
As you can see with issue #457, plugins states (warning/critical/unknown) aren't handled yet on the unstable version.
@steveschnepp is currently rewriting some parts of the engine behind this, so you'll have to wait a little bit before being able to use those.

If I'm not mistaken, demo uses the old alerts engine, that's why it still works.

TLDR; This is a known issue, since it isn't implemented yet

**Comment by chteuchteu** [comment]
@ssm: Can't this be closed since it is a duplicate of #457?

**Comment by sumpfralle** [comment]
yes, closing as a dumplicate of #457
